.github/workflows/tests.yaml

```diff
@@ -43,6 +43,9 @@ jobs:
           virtualenv --upgrade-embed-wheels
           # virtualenv --reset-app-data
           pip install -e .[tests]
+      - name: 🔍 Show installed package version
+        run: |
+          python -c "from importlib.metadata import version; print('Topostats package version:', version('topostats'))"
```

Comment:
A shorter way to get this is to use `pip show topostats`.

That said the log file should already tell you what the installed version of `topostats` is as the output of `pip install -e .[tests]` will report it.

Line [503](https://github.com/AFM-SPM/TopoStats/actions/runs/16933771643/job/47985359255?pr=1197#step:4:503) of the log file shows the same as this output (although you have to read through the list).

Weird tag and version!